api_server/src/http_service.rs

```diff
@@ -469,7 +469,10 @@ impl hyper::server::Service for ApiServerHttpService {
                 Ok(parsed_req) => match parsed_req {
                     GetInstanceInfo => {
                         METRICS.get_api_requests.instance_info_count.inc();
-
+                        info!(
+                            "The API server received a {}.",
```

Comment:
nit: this `info!(...)` call is identical throughout the file. You could move it to an inline helper function named smth like `log_api_request()` which takes the same params as `describe`.